code/controllers/subsystem/radiation.dm

```diff
@@ -0,0 +1,18 @@
+PROCESSING_SUBSYSTEM_DEF(radiation)
+	name = "Radiation"
+	flags = SS_NO_INIT | SS_BACKGROUND
+	wait = 1 SECONDS
+	offline_implications = "Radiation will no longer function; power generation may not happen. A restart may or may not be required, depending on the situation."
```

Comment:
Radiation components dont seem to be hooked into the SM, so this doesnt seem to affect power gen, though correct me if I am wrong

Reply:
the SM utilizes `radiation_pulse`, which won't work if this system is offlined, completely.